src/services/identity/index.ts

```diff
@@ -29,12 +29,6 @@ const OTP_EXPIRY = config.get("auth.otpExpiry")
 
 const IS_DEV = NODE_ENV === "dev"
 
-// TODO: To remove TOTP
-const totpGenerator = new TotpGenerator({
```

Comment:
May I get your help to remove the import `import TotpGenerator from "@services/identity/TotpGenerator"`, `const OTP_SECRET = config.get("auth.otpSecret")` and `const OTP_EXPIRY = config.get("auth.otpExpiry")`, and the corresponding convict stuff if its not being used in our codebase for a cleaner "cleanup"?

Reply:
sure, removed this and the corresponding env var in `config`